src/pages/signup/owner/steps/StepOperatingHours/ByDaySlots.jsx

```
@@ -0,0 +1,174 @@
+  if (!dayTimes || Object.keys(dayTimes).length === 0) {
+    return [{ days: [], times: [{ start: '09:00', end: '18:00' }] }];
+  }
+  const processed = new Set();
+  const groups = [];
+  Object.entries(dayTimes).forEach(([day, times]) => {
+    if (processed.has(day)) return;
+    const key = JSON.stringify(times);
+    const sameDays = Object.entries(dayTimes)
+      .filter(([d, t]) => !processed.has(d) && JSON.stringify(t) === key)
+      .map(([d]) => d);
+    sameDays.forEach((d) => processed.add(d));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)